### ISSUE TITLE
Support Labels, connected to inputs

### ISSUE THREAD

**Issue by theKashey** [jossmac/react-radios#1]
a11y 🤘Inputs should have corresponding labels. 
The current approach is not _"proper"_ way to do it.
Adding labels via `component` property is not the _"easy"_ way to do it.

**Comment by jossmac** [comment]
> The current approach is not "proper" way to do it.

What do you consider the "proper" way to do it?

> Adding labels via component property is not the "easy" way to do it.

What do you consider the "easy" way to do it?

**Comment by theKashey** [comment]
So - right now there is literally no way to connect any label to any checkbox or radio. The only way is to provide some component, which will render input+label as a `component` prop.
Probably doable, but this is something user shall not have to do every time.

I could see 2 ways to solve this:
 - expose `Label` component, which will connect to the corresponding input by known `value` prop.
 - expose `Pair` component, which will connect Input+Label tags inside it.
